src/shadowbox/infrastructure/prometheus_scraper.ts

```diff
@@ -86,7 +87,8 @@ export async function runPrometheusScraper(
   });
   const commandArguments = ['--config.file', configFilename];
   commandArguments.push(...args);
-  const runProcess = child_process.spawn('/root/shadowbox/bin/prometheus', commandArguments);
+  const sbBinDir = process.env.SB_BIN_DIR || '/root/shadowbox/bin';
```

Comment:
Globals are not great in libraries. Let's pass the binary location in the constructor, so `main()` can be in control of this logic.

Reply:
Done